snuba/datasets/errors_replacer.py

```diff
@@ -553,12 +577,29 @@ def process_exclude_groups(message: Mapping[str, Any]) -> Optional[Replacement]:
     See docstring in `sentry.reprocessing2` for more information.
     """
 
-    group_ids = message["group_ids"]
-    if not group_ids:
+    project_id: int
+    group_ids: Sequence[int]
+
+    @classmethod
+    def parse_message(
+        cls, message: Mapping[str, Any], context: ReplacementContext
```

Comment:
Why passing `context` in here?

Reply:
In the next steps this is about to become a common interface across all replacements, and every replacement has the same `parse_message`. This also allows me to refactor routing. See the big WIP-PR for where I want to go with this.